ui/Screen/Profile/Projects.svelte

```diff
@@ -94,8 +96,20 @@
         </div>
       </Flex>
     </List>
+  {:else if session.permissions.registerHandle}
+    <EmptyState
+      text="There’s nothing here yet, get started by starting your first
+      project."
+      primaryActionText="Start your first project"
+      secondaryActionText="Or register your handle"
+      on:primaryAction={create}
+      on:secondaryAction={register} />
   {:else}
-    <Onboard />
+    <EmptyState
+      text="There’s nothing here yet, get started by starting your first
+      project."
+      primaryActionText="Or register your handle"
```

Comment:
This should be "Create a new project".

Reply:
addressed in [7ad5dc2](https://github.com/radicle-dev/radicle-upstream/pull/537/commits/7ad5dc27b7afa4429abe1f5ed60cd4531358614a)